### PR TITLE
Link to dashboard

### PR DESCRIPTION
**Pull request by CamLatimer**
I rule and have completed some work on Case Manager that's ready for review!

This pull request makes the following changes:
* Adds link in navigation partial view so logged in users can get back to the dashboard.  If the user is already at the dashboard view, the link is not rendered.

It relates to the following issue #s: 
* Issue #248

### CODE REVIEW

**Comment by colinxfleming** [comment]
💯 for using the template! Marking up in a sec.

File: app/views/layouts/_navigation_links.html.erb [hunk 1]
```diff
@@ -1,7 +1,11 @@
+
 <ul class="nav navbar-nav navbar-right">
   <% if !user_signed_in? %>
     <li><%= link_to 'Sign in', new_user_session_path %></li>
   <% else %>
+    <% unless current_page?(authenticated_root_url) %>
+      <p class="navbar-text"><%= link_to 'Dashboard', authenticated_root_url %></p>
```
Comment (colinxfleming) [inline app/views/layouts/_navigation_links.html.erb]: Nicely done! Clean and efficient, love it.

**Comment by colinxfleming** [comment]
Looks good to me! I'll post a screenshot in this thread on Friday and then merge.

**Comment by CamLatimer** [comment]
Thanks, Colin. I'm also aiming to work on issues outside of meetup days.
This next week and a half I won't be able to give it much attention, but
I'll have lots of time after this month.

**Comment by colinxfleming** [comment]
Sounds good, I'll plan accordingly. Thanks again Cam!

**Comment by colinxfleming** [comment]
Changing the issue to an integration test to confirm behavior.

**Pull request merged**